core/station/impl/src/models/request_policy.rs

```diff
@@ -53,10 +53,29 @@ impl From<MatchError> for EvaluateError {
     }
 }
 
+// metadata related rules can only be applied to accounts
```

Comment:
A generic remark: should we avoid wildcards to ensure we for sure catch all rules and specifiers (also in the future)?